src/coreclr/tests/src/JIT/opt/InstructionCombining/ArrayLengthArithmetic.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+using System.Runtime.CompilerServices;
+
+public class ArrayLengthArithmeticTests
+{
+    static int returnCode = 100;
```

Comment:
nit: consider following the code style guidelines here (including adding explicit access modifiers).